types/index.d.ts

```diff
@@ -1,14 +1,14 @@
 export = dayjs;
 
-declare function dayjs (config?: dayjs.ConfigType, option?: dayjs.OptionType): dayjs.Dayjs
+declare function dayjs (date?: dayjs.DateType, option?: dayjs.OptionType): dayjs.Dayjs
 
 declare namespace dayjs {
-  export type ConfigType = string | number | Date | Dayjs
+  export type DateType = string | number | Date | Dayjs
```

Comment:
this could be a breaking change for those, who use TS and rely on semver. I suppose it is better to increase at least minor version, or to not change naming right now at all, but leave comments that it is deprecated and will be replaced in newer versions

Reply:
Added type alias with `@deprecated` marker!